data/hai/hai missions.txt

```diff
@@ -341,11 +360,20 @@ mission "Expanding Business [1]"
 	destination "Allhome"
 	to offer
 		has "event: hai-human resolution announced"
+		or
+			not "hai start"
+			"combat rating" > 500
 	on offer
 		conversation
 			`Walking through the <origin> spaceport, you notice that almost half of the people you see are humans, unlike other Hai worlds where the Hai outnumber humans by a sizable degree. A human dressed in a business suit shouts your name from across the street and approaches you.`
+			branch "hai start"
+				has "hai start"
 			`	"Captain <last>, fantastic to see you again! I noticed you right away when I laid my eyes upon you and I must say, I'd like to thank you again for what you've done."`
 			`	The man is David Turner, who was involved in the movie production during the Hai crisis. Almost reflexively he goes to grab a business card from his pocket but then seems to remember and puts it back away. Seems to be exactly the same card as last time though.`
+			label "hai start"
+			`	"Hello, captain. I come to you with a business opportunity you would do well to listen to." He grabs his business card to show it to you, and presents himself. "My name is David Turner, you may know me for my involvement with the Hai cover movie? I come to you because you seem to have quite a bit of combat experience, which would be of great help for this business venture." He then adds something, in a low tone, "Also, you are not a pirate saying that they redeemed themselves..."`
```

Comment:
```suggestion
			`	"Hello, captain. I come to you with a business opportunity you would do well to listen to." He grabs his business card to show it to you and presents himself. "My name is David Turner, you may know me for my involvement with the Hai cover movie? I come to you because you seem to have quite a bit of combat experience, which would be of great help for this business venture." He then adds, in a low tone, "Also, you are not a pirate saying that they redeemed themselves...`
```